### ISSUE TITLE
[BUG] Export API contains more than application names

### ISSUE THREAD

**Issue by shan916** [Axway-API-Management-Plus/apimanager-swagger-promote#186]
**Swagger-Promote Version**
1.6.3-1 introduces this bug. 1.6.3 did not have this

**API-Manager and Service-Pack Version**
7.5.3 SP7

**Expected behavior**
When exporting an API, the export api-config.json should contain just the application names like:
```
"applications": [
    {
      "name": "application1"
    },
    {
      "name": "application2"
    }
]
```

**Actual behavior**
The actual output of api-config.json is:
```
"applications": [
    {
        "name": "application1",
        "apiAccess": [
            {
                "id": "xxx",
                "apiId": "yyy",
                "createdBy": "zzz",
                "state": "approved",
                "createdOn": "1508278834047",
                "enabled": true
            }
        ]
    },
    {
        "name": "application2",
        "apiAccess": [
            {
                "id": "xxx",
                "apiId": "yyy",
                "createdBy": "zzz",
                "state": "approved",
                "createdOn": "1554941893214",
                "enabled": true
            }
        ]
    }
]
```

**Test case sample**
Run an export on API that is granted access to some application(s). Run with Swagger Promote v1.6.3 and v1.6.3-1 and compare the `applications` block in api-config.json

**Comment by cwiechmann** [comment]
Have realized, that my integration tests have not covered the export of applications. Created a new integration tests for that that validates only the name-field is exported and fixed the issue.

**Comment by cwiechmann** [comment]
Fixed with release 1.6.3-2
